src/components/organisms/ChatSidebar/components/RecipientChat/RecipientChat.tsx

```diff
@@ -19,10 +19,12 @@ export const RecipientChat: React.FC<RecipientChatProps> = ({
 }) => {
   const {
     messagesToDisplay,
+    recipient,
+
     sendMessageToSelectedRecipient,
     deleteMessage,
     markMessageRead,
-    recipient,
+    sendThreadReply,
   } = useRecipientChat(recipientId);
```

Comment:
I think the answer to this is because there are different types of chat that can be sent, yet they all share the same `Chatbox` (but different send functions)

Reply:
> I think the answer to this is because there are different types of chat that can be sent, yet they all share the same Chatbox (but different send functions)

Yes